### PR TITLE
WIP authors: split blueprints

### PR DESCRIPTION
**Pull request by jacenkow**
* Split author module view into holdingpen and publications views.

* Adds basic test to prevent regression.

Co-authored-by: Jacopo Notarstefano <jacopo.notarstefano@cern.ch>
Signed-off-by: Grzegorz Jacenków <grzegorz.jacenkow@cern.ch>

### CODE REVIEW

**Comment by jacenkow** [comment]
CC: @jacquerie

File: tests/integration/test_author_views.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        assert client.get(
+            '/author/publications?recid=984519').status_code == 200
+
+
+def test_author_publications_returns_the_right_data(app):
+    with app.test_client() as client:
+        response = json.loads(
+            client.get('/author/publications?recid=984519').data)
+
+        # FIXME: Assert should be more specific.
```
Comment (jacquerie) [inline tests/integration/test_author_views.py]: Nit: `# FIXME: asserts should be more specific.`

**Comment by jacquerie** [comment]
### TODO:

- [ ] We fail due to a bundling error (https://travis-ci.org/inspirehep/inspire-next/jobs/129155669#L1264). In fact, we are still declaring that our templates and static files are at the same level as the blueprint, which is no longer the case. This should be an easy fix for tomorrow.
- [ ] Add a commit to fix the conference view issue we have been seeing for a while (@eamonnmag).
- [ ] Add a commit to delete ES data in the integration fixture, so that we don't get bogus failures.

When this is done, I think we can 🚢 this PR.

File: tests/integration/conftest.py [hunk 2]
```diff
@@ -56,6 +56,7 @@ def teardown():
 
         sleep(10)  # Makes sure that ES is up.
         _es = app.extensions['invenio-search']
+        list(_es.delete(ignore=[404]))
```
Comment (jacquerie) [inline tests/integration/conftest.py]: Move these changes to the previous commit, because they belong there.

File: tests/integration/conftest.py [hunk 3]
```diff
@@ -56,6 +56,7 @@ def teardown():
 
         sleep(10)  # Makes sure that ES is up.
         _es = app.extensions['invenio-search']
+        list(_es.delete(ignore=[400]))
```
Comment (jacquerie) [inline tests/integration/conftest.py]: The problem with doing this is that, if you `git bisect`, you get a broken build on this commit. Ideally we should be passing the suite at every commit!

File: tests/integration/conftest.py (hunk 2 above)
Comment (jacenkow) [inline tests/integration/conftest.py]: Done.

Comment (jacquerie) [inline tests/integration/conftest.py]: Thanks!

**Comment by jacquerie** [comment]
Commit 2479f6f should be replaced with https://github.com/jacquerie/inspire-next/commit/f56e5810cdf0044d745581d11cb652ef699a8e52 due to an error found by @jmartinm. Then, after a rebase, we :shipit:.

**Comment by jacquerie** [comment]
Superseded by #1112.

**Pull request closed without merging**